rero_ils/modules/documents/mappings/v7/documents/document-v0.0.1.json

```diff
@@ -2116,6 +2116,9 @@
       "status": {
         "type": "keyword"
       },
+      "isFiction": {
```

Comment:
isFiction or `fiction` , harvested and fiction should have the same naming or nomenclature.

Reply:
changed to fiction